package.json

```diff
@@ -3,43 +3,73 @@
   "version": "0.5.2",
   "description": "Badges for Formidable OSS",
   "license": "MIT",
+  "types": "dist/src/index.d.ts",
```

Comment:
Seems weird to me to have `dist/src` –  I'd expect `src` be like the "root" and map directly into `dist`. (I'm sure it's just like a TS/build config, and doesn't really matter – just caught me a bit off guard.)

Reply:
@gksander I believe it just mirrors the source structure for outputting types. Since `index.ts` is in `src`, `vite-plugin-dts` will just produce type files at the same dir structure.

Reply:
ah makes sense! dope.